src/ActionRunner/Resources.resx

```diff
@@ -59,7 +59,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="DOTNET_CORE_DOWNLOAD_FAILURE" xml:space="preserve">
```

Comment:
The **uid** names might be updated aswel? :)

Reply:
You mean DOTNET_CORE_DOWNLOAD_FAILURE?

Reply:
Yeah, I believe.